webapp/channels/src/components/modal_controller/modal_controller.tsx

```diff
@@ -5,8 +5,8 @@ import React from 'react';
 
 type Modal = {
     open: boolean;
-    dialogProps: Record<string, any>;
     dialogType: React.ComponentType;
+    dialogProps?: Record<string, any>;
```

Comment:
not related to this change, but was just showing a warning in the console that from some places the dialogProps was not being send because wasn't necessary, so making this one optional.

Reply:
Makes sense to me. We already have `modal.dialogProps?.` below so we've likely ran into error cases where this wasn't set before